common/to.etc.alg/src/main/java/to/etc/util/DateUtil.java

```diff
@@ -55,14 +55,16 @@
 	@NonNull
 	private final static ThreadLocal<Calendar> m_testDate = new ThreadLocal<>();
 
+	public static final String DATE_MASK = "yyyyMMdd";
```

Comment:
This does not belong here; it does not mean anything. You use it in your Excel fix where it DOES seem to mean something - namely a format Excel uses as its default format. So it belongs there, not here.